docs/src/assets/chainrules.css

```
@@ -0,0 +1,78 @@
+/* Links */
+
+a {
+  color: #4595D1;
+}
+
+a:hover, a:focus {
+  color: #194E82;
+}
+
+/* Navigation */
+
+nav.toc ul a:hover,
+nav.toc ul.internal a:hover {
+  color: #FFFFFF;
+  background-color: #4595D1;
+}
+
+nav.toc ul .toctext {
+  color: #FFFFFF;
+}
+
+nav.toc {
+  box-shadow: none;
+  color: #FFFFFF;
+  background-color: #194E82;
+}
+
+nav.toc li.current > .toctext {
+  color: #FFFFFF;
+  background-color: #4595D1;
+  border-top-width: 0px;
+  border-bottom-width: 0px;
+}
+
+nav.toc ul.internal a {
+  color: #194E82;
+  background-color: #FFFFFF;
+}
+
+/* Text */
+
+article#docs a.nav-anchor {
+  color: #194E82;
+}
+
+article#docs blockquote {
+  font-style: italic;
+}
+
+/* Terminology Block */
+
+div.admonition.terminology div.admonition-title:before {
+  content: "Terminology: ";
+  font-family: inherit
```

```suggestion
  font-family: inherit;
```
This was making it render weird on my laptop.